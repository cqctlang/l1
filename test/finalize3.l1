// no output: the point of this test is not to crash or mishandle memory
for(i = 0; i < 20000; i++){
	finalize(i, @lambda(val) { return 1; });
}
