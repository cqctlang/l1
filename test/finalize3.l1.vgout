entry: var params, 1 locals, 0 temps
L9: 1 params, 0 locals, 0 temps
