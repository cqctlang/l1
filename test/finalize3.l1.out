entry: var params, 2 locals, 0 temps
L11: 1 params, 0 locals, 0 temps
